packages/pages/src/util/viteConfig.ts

```diff
@@ -16,3 +16,12 @@ export const scopedViteConfigPath = (scope?: string) => {
     return viteConfigPath;
   }
 };
+
+export const removePluginFromViteConfig = (config: any) => {
+  if (config?.plugins) {
+    config.plugins = config.plugins?.[0]?.filter(
+      (obj: any) => obj.name !== "vite:react-refresh"
```

Comment:
If I console.log it, vite.config.js has two plugins 
```
  "plugins": [
    [
      {
        "name": "vite:react-babel",
        "enforce": "pre"
      },
      {
        "name": "vite:react-refresh",
        "enforce": "pre"
      }
    ],
    {}
  ],
```

my vite.config.js looks like:
```
export default defineConfig({
  resolve: {
    alias: {
      src: path.resolve("src/"),
    },
  },
  plugins: [react(), yextSSG()],
  build: {
    rollupOptions: {
      external: ['ssh2']
    }
  }
});
```

upon removing the `vite:react-refresh` plugin, the infinite looping doesn't occur. will look into exactly why tmrw.

Reply:
talked offline.